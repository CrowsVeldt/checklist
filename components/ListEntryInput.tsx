import { StyleSheet, TextInput, View } from "react-native";

export default function ListEntryInput() {
  return (
    <View style={styles.inputWrapper}>
      <TextInput defaultValue="Entry Title" />
    </View>
  );
}

const styles = StyleSheet.create({
  inputWrapper: {
    flexDirection: "row",
    alignItems: "center",
  },
});
