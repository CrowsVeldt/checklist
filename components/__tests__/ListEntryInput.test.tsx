import { render } from "@testing-library/react-native";
import ListEntryInput from "../ListEntryInput";

describe("<ListEntryInput />", () => {
  test("ListEntryInput renders correctly", () => {
    const tree = render(<ListEntryInput />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
