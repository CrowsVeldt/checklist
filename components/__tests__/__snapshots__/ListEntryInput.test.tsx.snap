// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ChecklistItem /> ListEntryInput renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flexDirection": "row",
    }
  }
>
  <TextInput
    defaultValue="Entry Title"
  />
</View>
`;
